smart-contracts/contracts/CosmosBridge.sol

```diff
@@ -204,7 +180,8 @@ contract CosmosBridge is CosmosBridgeStorage, Oracle {
     view
     returns (uint256 pow)
   {
```

Comment:
I couldn't find any unit tests for this function?

Reply:
Its private so we can't call it from the test scripts, it gets tested when batchSubmitProphecyClaimAggregatedSigs is tested

Reply:
I don't think theres tests to verify duplicate signers would trigger revert

Reply:
you available on slack to huddle?

Reply:
See testSignatureAggregationSecurity.js
```js
    it("there are duplicate signers", async function () {
      state.recipient = userOne;
      state.nonce = 1;

      const { digest, claimData, signatures } = await getValidClaim({
        sender: state.sender,
        senderSequence: state.senderSequence,
        recipientAddress: state.recipient.address,
        tokenAddress: state.troll.address,
        amount: state.amount,
        doublePeg: false,
        nonce: state.nonce,
        networkDescriptor: state.networkDescriptor,
        tokenName: state.name,
        tokenSymbol: state.symbol,
        tokenDecimals: state.decimals,
        cosmosDenom: state.constants.denom.none,
        validators: [userOne, userTwo, userFour, userFour],
      });

      await expect(
        state.cosmosBridge
          .connect(userOne)
          .submitProphecyClaimAggregatedSigs(digest, claimData, signatures)
      ).to.be.revertedWith("DUP_SIGNER");
    });
    ```